### ISSUE TITLE
JWT Crash Prohibits Login

### ISSUE THREAD

**Issue by nwself** [NGS2Montage/montage#8]
Seems to crash here: `montage_jwt/util.py", line 14, in make_login_token` 

I have `private.pem` at the top level (with README.md, requirements.txt, etc.). I'm guessing my settings/configuration is wrong somehow. But I followed the README.

To reproduce:
- `./manage.py createsuperuser` then follow prompts
- `./manage.py runserver`
- Browse to site, try to login

```
[14/Sep/2017 13:11:15] "POST /user/login/ HTTP/1.1" 302 0
Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nwself/.virtualenvs/montage/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/Users/nwself/dev/montage/montage_jwt/middleware.py", line 54, in __call__
    new_jwt = make_login_token(request.user)
  File "/Users/nwself/dev/montage/montage_jwt/util.py", line 14, in make_login_token
    jwt = JWT.objects.create_token(claims, user)
  File "/Users/nwself/dev/montage/montage_jwt/models.py", line 16, in create_token
    token = jwt.encode(claims, api_settings.PRIVATE_KEY, algorithm=api_settings.ALGORITHM).decode('utf-8')
  File "/Users/nwself/.virtualenvs/montage/lib/python3.6/site-packages/jwt/api_jwt.py", line 56, in encode
    json_payload, key, algorithm, headers, json_encoder
  File "/Users/nwself/.virtualenvs/montage/lib/python3.6/site-packages/jwt/api_jws.py", line 102, in encode
    key = alg_obj.prepare_key(key)
  File "/Users/nwself/.virtualenvs/montage/lib/python3.6/site-packages/jwt/algorithms.py", line 209, in prepare_key
    raise TypeError('Expecting a PEM-formatted key.')
TypeError: Expecting a PEM-formatted key.
[14/Sep/2017 13:11:15] "GET / HTTP/1.1" 500 77651
```

**Comment by mttwong** [comment]
I'll take a look.

**Comment by mttwong** [comment]
Should be fixed at 560fbfa1de7776fb4f1211efb79e386a6a2745fc.  The issue is in local_settings.py.  Will add more documentation for settings tomorrow.
